infrastructure/lambda/template.yaml

```diff
@@ -460,13 +422,6 @@ Resources:
             Action:
```

Comment:
Do we really want to be granting access to ALL of KMS functionality for ALL resources (regardless of region, stack, etc?)

I think this should at least be locked down to the stack or at very least environment